docker-compose.yml

```
@@ -31,75 +31,52 @@ services:
       - DB_NAME=${DB_NAME}
       - DB_USER=${DB_USER}
       - DB_PASSWORD=${DB_PASSWORD}
-      - REMOTE_USER_AUTHENTICATION=True
+      - REMOTE_USER_AUTHENTICATION=False
       - SITE_ROOT=api
       - SITE_URL=${NEW_OPENIMIS_HOST}
-      - CELERY_BROKER_URL=amqp://rabbitmq
+      - CELERY_BROKER_URL=amqp://localhost
```

to send the config at runtime too
```suggestion
      - CELERY_BROKER_URL=amqp://localhost
      - OPENIMIS_CONF_JSON=${OPENIMIS_BE_CONF_JSON}
```